src/main/xsl/preprocess/maprefImpl.xsl

```diff
@@ -389,6 +394,23 @@ See the accompanying LICENSE file for applicable license.
     </xsl:call-template>
   </xsl:template>
 
+  <xsl:template match="@*" mode="preserve-submap-attributes">
+    <xsl:attribute name="dita-ot:submap-{local-name()}" select="."/>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class,' topic/title ')]" mode="preserve-submap-title-and-topicmeta">
+    <dita-ot:submap-title class="+ topic/foreign ditaot-d/submap-title ">
```

Comment:
Since DITA has at least traditionally been anti-namespace, I think we could just use `<submap-title class="+ topic/foreign ditaot-d/submap-title ">` here. That's what we've been doing in other places: elements in empty namespace and attribute in DITA-OT namespace.